lib/restforce/db/tracker.rb

```diff
@@ -14,9 +14,9 @@ class Tracker
       #
       # file_path - The Path to the tracking file.
       def initialize(file_path)
-        @file = File.open(file_path, "a+")
+        @file_path = file_path
 
-        timestamp = @file.read
+        timestamp = File.open(@file_path, "a+") { |file| file.read }
```

Comment:
Why `a+` here, if we're just reading? Is this to cause the file to exist if it doesn't?